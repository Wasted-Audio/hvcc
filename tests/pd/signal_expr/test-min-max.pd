#N canvas 827 239 734 565 12;
#X obj 60 47 loadbang;
#X obj 60 74 bng 25 250 50 0 empty empty empty 17 7 0 10 #191919 #ffffff #ffffff;
#X obj 60 437 print;
#X obj 74 264 sig~;
#X obj 60 405 snapshot~;
#X obj 60 108 t_delays, f 37;
#X msg 107 139 2.3;
#X obj 202 264 sig~;
#X obj 202 437 print;
#X obj 202 405 snapshot~;
#X msg 202 139 -3.1;
#X msg 298 139 5.7;
#X obj 74 298 expr~ min($v1 \, 5);
#X obj 202 298 expr~ max($v1 \, 5);
#X obj 298 172 t f f;
#X obj 250 172 t b b;
#X obj 202 172 t f f;
#X obj 155 172 t b b;
#X obj 107 172 t f f;
#X obj 60 172 t b b;
#X obj 60 357 del 1;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 3 0 12 0;
#X connect 4 0 2 0;
#X connect 5 0 19 0;
#X connect 5 1 6 0;
#X connect 5 2 17 0;
#X connect 5 3 10 0;
#X connect 5 4 15 0;
#X connect 5 5 11 0;
#X connect 6 0 18 0;
#X connect 7 0 13 0;
#X connect 9 0 8 0;
#X connect 10 0 16 0;
#X connect 11 0 14 0;
#X connect 12 0 4 0;
#X connect 13 0 9 0;
#X connect 14 0 3 0;
#X connect 14 1 7 0;
#X connect 15 0 20 0;
#X connect 15 1 9 0;
#X connect 16 0 3 0;
#X connect 16 1 7 0;
#X connect 17 0 20 0;
#X connect 17 1 9 0;
#X connect 18 0 3 0;
#X connect 18 1 7 0;
#X connect 19 0 20 0;
#X connect 19 1 9 0;
#X connect 20 0 4 0;
