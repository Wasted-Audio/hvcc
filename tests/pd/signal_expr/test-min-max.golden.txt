[@ 2.021] print: 5.7
[@ 3.021] print: 5
[@ 6.021] print: 5
[@ 7.021] print: -3.1
[@ 10.021] print: 5
[@ 11.021] print: 2.3
